operations/node_management/Adding_a_Liquid-cooled_blade_to_a_System.md

```diff
@@ -376,6 +376,43 @@ This procedure will add a liquid-cooled blades from an HPE Cray EX system.
             curl -k -u root:password https://x1005c3s0b0/redfish/v1/Managers
             ```
 
+1. Clear out the existing Redfish event subscriptions from the BMCs on the blade.
+
+    1. (`ncn#`) Set the environment variable `SLOT` corresponding to the blades location:
+
+        ```bash
+        SLOT="x1005c3s0"
+        ```
+
+    1. (`ncn#`) Clear the Redfish event subscriptions:
+
+        ```bash
+        for BMC in $(cray hsm inventory  redfishEndpoints list --type NodeBMC --format json | jq .RedfishEndpoints[].ID -r | grep $SLOT); do
+            PASSWD=$(cray scsd bmc creds list --targets $BMC --format json | jq .Targets[].Password -r)
+            SUBS=$(curl -sk -u root:$PASSWD https://${BMC}/redfish/v1/EventService/Subscriptions | jq -r '.Members[]."@odata.id"')
```

Comment:
Would be better to avoid putting the password on the curl command line, because that is visible in the output of `ps -ef` for any users logged into the system, regardless of access level.

Reply:
Do you have a suggestion on how to do this? Or another place in the docs where this is done?

Reply:
The way I've done it before in similar cases is to export the password as an environment variable, and then make a tiny Python script which just makes the API request using it, and prints out the body contents.

Of course, given that you're piping to jq and parsing the output, and reusing this code in several places, it may be better to replace more (or all) of the code with a Python script.

I did this a long time ago in the NTP/DNS fixup script that Jacob wrote. I can find a link to it. I think there are other places in the docs repo where I've done something similar (or others have done so). I'll see if I can find some links...